packages/pagination/src/Pagination.types.ts

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+
+export type CustomOnClick = (page: number) => void;
+
+export interface PaginationProps
+  extends Omit<React.HTMLAttributes<HTMLElement>, 'onClick'> {
+  /**
+   * `className` - additional styles, like color
+   */
+  className?: string;
+  /**
+   * `totalPages` - total number of pages
+   */
+  totalPages: number;
+  /**
+   * `currentPage` - current page
+   */
+  currentPage: number;
+  /**
+   * `renderItem` - template function to allow different components and props for PaginationItem
+   */
+  renderItem?: (item: PaginationItemProps) => JSX.Element;
+  /**
+   * `onClick` - template function for onClick event of items
+   */
+  onClick?: CustomOnClick;
+  /**
+   * `aria-label` - aria-label for root pagination component
+   */
+  'aria-label'?: string;
+  /**
+   * `getItemAriaLabel` - Function for generation of aria-label of clickable items
+   */
+  getItemAriaLabel?: getItemAriaLabelFunction;
+}
+
+export interface DefaultGetItemAriaLabelProps {
+  type: 'page' | 'next' | 'previous';
+  page: number;
+  selected: boolean;
+}
+
+export type getItemAriaLabelFunction = ({
+  type,
+  page,
+  selected,
+}: DefaultGetItemAriaLabelProps) => string;
+
+export enum ariaLabelType {
+  page = 'page',
+  previous = 'previous',
+  next = 'next',
+}
+
+export enum paginationItemType {
+  page = 'page',
```

Comment:
is this the page number? if yes, can we rename it as `pageNumber` please?

Reply:
Ok